apps/gallery/test/marionette/lib/fullscreen_view.js

```diff
@@ -255,7 +256,31 @@
         Fullscreen_View.Selector.shareButton);
     },
 
-      /**
+    /**
+     * @return {Marionette.Element} Element containing fullScreenView toolbar
+     */
+    get fullScreenToolBar() {
+      return this.client.helper.waitForElement(
+        Fullscreen_View.Selector.fullScreenToolBar);
+    },
+
+    /**
+    * @return {numeric} fullScreenToolBar width.
+    */
+    get ToolBarWidth() {
+      return this.fullScreenToolBar.size().width;
+    },
+
+    /**
+     * @return {Marionette.Element}  List of elements of toolbar options.
+     */
+    get ToolBarOptions() {
+      this.waitFor(Fullscreen_View.Selector.fullScreenToolBarOptions);
```

Comment:
No specific preference here, I guess I am using it as its available :), initially it was exposed to abstract repeated calls while writing image edit tests.